src/test/java/com/google/firebase/auth/TenantAwareFirebaseAuthIT.java

```diff
@@ -61,20 +64,21 @@
   private static final JsonFactory jsonFactory = Utils.getDefaultJsonFactory();
   private static final HttpTransport transport = Utils.getDefaultTransport();
 
-  private static FirebaseAuth auth;
   private static TenantManager tenantManager;
-  private static TenantAwareFirebaseAuth tenantAwareAuth;
+  private static TenantAwareFirebaseAuth auth;
```

Comment:
Nit: I kindda liked the more descriptive name `tenantAwareAuth` here. But your call.

Reply:
Done.